doc/changelog.md

```diff
@@ -2,6 +2,53 @@
 
 <h3>New features</h3>
 
+* Support for disabling Autograph for a specific function.
+[(#705)](https://github.com/PennyLaneAI/catalyst/pull/705)
+
+  The following is now possible:
+
+```py
```

Comment:
It would be nice to indent everything in the changelog entry

Reply:
Didn't realize, sorry. Done now :)